.github/workflows/performance.yml

```
@@ -0,0 +1,27 @@
+name: "Performance"
```

Rename workflow file to `performance-tests.yml`, so its more clear what it is about.

```suggestion
name: "Performance Tests"
```